waku/persistence/postgres/mock.go

```diff
@@ -0,0 +1,51 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+)
+
+// var dbUrlTemplate = "postgres://postgres@localhost:%s/%s?sslmode=disable"
+var dbUrlTemplate = "postgres://harshjain@localhost:%s/%s?sslmode=disable"
```

Comment:
Pretty sure you don't want to leave this like this. Also, shouldn't the host and the user be also templateable?